app/models/event_stream.rb

```diff
@@ -35,13 +35,21 @@ class EventStream < ApplicationRecord
 
   after_commit :emit_notifications, :on => :create
 
-  # TODO: Consider moving since this is EmsEvent specific. group, group_level and group_name exposed as a virtual columns for reports/api.
-  GROUP_LEVELS = %i(critical detail warning).freeze
+  DEFAULT_GROUP_NAME  = :other
+  DEFAULT_GROUP_LEVEL = :detail
```

Comment:
Similarly, for i18n here, you'd do something like:

```suggestion
  DEFAULT_GROUP_NAME  = N_("Other")
  DEFAULT_GROUP_LEVEL = N_("Detail")
```

Then later or perhaps in another constant you'd do `DEFAULT_GROUP_NAME.downcase.to_sym`